ccd-definition/CaseEventToComplexTypes/returnApplication/ReturnApplication.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ReturnApplication",
+    "CaseEventID": "returnApplication",
+    "CaseFieldID": "returnApplication",
+    "ListElementCode": "reason",
```

Comment:
are these names a bit generic, do we want to go with returnReason or what is your thoughts? I'm thinking outside the context of this json file it might be a bit unclear..

Reply:
No I think they are fine. I'd prefer 

ReturnApplication.getReason() over ReturnApplication.getReturnReason(); 

I think the later is unnecessarily verbose.

Reply:
cool makes sense. :)